docs/learn/learn-elastic-scaling.md

```diff
@@ -75,11 +67,52 @@ The diagram above shows how four parablocks are backed and included in the relay
 cores (C1 and C2). Note how P2 and P4 are included after 18 seconds (not 12 seconds) because when P1
 and P3 were pushed to the relay chain for being backed, there were no other parablocks before them.
 
-## Technical Considerations
+The relay chain receives a sequence of parachain blocks on multiple cores, which are validated and
+checked if all their state roots line up during their inclusion, but assume they’re unrelated
+parachain blocks during backing, availability, and approvals. With elastic scaling implemented, a
+parachain's throughput depends upon its collator infrastructure.
 
 If the pace per core on the relay chain will not change (backing and inclusion every 6 seconds per
 core), on the parachain side, collators will need to increase the parablock production rate to push
-P1 and P2 to the two relay chain cores.
+P1 and P3 to the two relay chain cores.
+
+## Elastic Scaling Benefits
+
+- **Higher throughput**: Parachains can utilize multiple cores simultaneously, enabling much faster transaction processing. Previously, parachains were limited to one block per Relay Chain block. They can handle much greater capacity with elastic scaling, significantly increasing overall performance.
```

Comment:
With the exception that they have powerful enough collators that are able to process more transactions (and subsequently, make and process more blocks) faster ;)

Reply:
Yes, this is specified. I will unify the sections where it is mentioned ;)